DDCore/include/DD4hep/IOV.h

```diff
@@ -68,20 +70,16 @@ namespace dd4hep {
     explicit IOV() = delete;
   public:
     /// Key definition
-    typedef long Key_first_type;
-    typedef long Key_second_type;
-    typedef std::pair<Key_first_type,Key_second_type> Key;
+    using Key_value_type = std::uint64_t;
```

Comment:
I am not sure if this change from "long" to "unsigned long" actually causes the problems in the tests.

Reply:
I changed it to `std::int64_t`, and I'll prepare another PR to remove the sign.